system/jlib/jtrace.hpp

```diff
@@ -273,8 +274,10 @@ interface ITraceManager : extends IInterface
     virtual bool isTracingEnabled() const = 0;
  };
 
- //Create an internal span that has now completed, but which started in the past.
- extern jlib_decl ISpan * createBackdatedInternalSpan(const char * name, stat_type elapsedNs);
+//Create an internal span that has now completed, but which started in the past.
+extern jlib_decl ISpan * createBackdatedInternalSpan(const char * name, stat_type elapsedNs);
```

Comment:
Wouldn't a backdated span create need a start time as well ?

Reply:
ok, understood.
Might just be leading blank spaces on these two lines are different ?